src/AClient.cpp

```diff
@@ -31,6 +31,11 @@ const std::string& AClient::getUser( void ) const { return( _user ); }
 
 const std::string& AClient::getIp( void ) const { return ( _ip ); }
 
+const std::string AClient::getOrigin( void ) const {
+    std::string const origin = ":" + getNick() + "!" + getUser() + "@" + getIp() + " ";
```

Comment:
why are we adding a space at the end? this should be trimmed to prevent problems in the future, we will add spaces when needed at the time of usage.